requirements-release.txt

```diff
@@ -1,4 +1,4 @@
-Django>=1.8,<1.9
+Django>=1.11.19
```

Comment:
I'll research what we need for this. As is this would mean it will install the latest version, which right now is 2.2.7, and I'm not sure that would work with this script.